components/harvestTx.ts

```diff
@@ -15,30 +15,38 @@ if (!BLOCKFROST_URL || !BLOCKFROST_KEY || !NETWORK) {
   throw 'Environment variables not set'
 }
 
-type HarvestReqBody = {
+type HarvestReqBody = [{
   poolIndex: number,
   address: string
-}
+}]
 
-const harvest = async (prisma: PrismaClient, body: string) => {
+const harvestCtrl = async (prisma: PrismaClient, body: string) => {
 
   const parsedBody: HarvestReqBody = JSON.parse(body)
-  let poolIndex = parsedBody.poolIndex
-  let address = parsedBody.address
-  console.log(`Body: ${body}`)
+  console.log(parsedBody)
+  const harvestTxRes = await Promise.all(parsedBody.map(async (harvestReq) => {
+    return await harvestTX(prisma, harvestReq);
+  }));
```

Comment:
We should send all 3 tokens in 1 transaction and not 3 transactions so that the user in the frontend only needs to sign 1 message instead of 3 messages (better UX).